libgphoto2/gphoto2-abilities-list.c

```diff
@@ -92,6 +92,7 @@ gp_message_codeset (const char *codeset)
  * relevant gettext text domain(s).
  *
  * \param localedir Root directory of libgphoto2's localisation files.
+ *                  If NULL, the default localedir will be locked in.
```

Comment:
What does it mean - _locked in_?

Reply:
Good point. That can be phrased better.

I meant to write

    If NULL, the default localedir value will be used, but you cannot
    call gp_init_localedir() again to change from the default value to
    another value.

However, this makes me wonder about the function's behaviour if someone calls it a second time. If that second call comes from inside  `gp_abilities_list_new()`, we want to ignore it. If that second call comes from someone else's code and the given `localedir` value differs from the value given earlier, we might want to give them a hint that this call failed.

I guess we could
  * document that the first call to `gp_init_localedir()` always wins
  * `localedir` value `NULL` means "use the default value"
  * all calls to `gp_init_localedir()` after the first call are ignored
  * log the successful call as being successful
  * possibly log subsequent calls as being too late

Reply:
https://github.com/gphoto/libgphoto2/pull/796/commits/820e6e7211a5af267d98c9776a94f99c922265ba should fix this